docs/configuration/entrypoints.md

```diff
@@ -120,6 +120,14 @@ To redirect an entrypoint rewriting the URL.
 !!! note
     Please note that `regex` and `replacement` do not have to be set in the `redirect` structure if an entrypoint is defined for the redirection (they will not be used in this case).
 
+`regex` and `replacement` follow the rules as per [matchers](/basics/#matchers)
```

Comment:
This is false, because in matchers we use goriila/mux regex, it's not the same syntax.

Could you remove this sentence.

Reply:
Oh thats interesting, have added your extra comment about how they are different